src/marketplace/sellers/mftf-magento.md

```diff
@@ -0,0 +1,56 @@
+---
+group: marketplace-sellers
+title: MFTF Magento Supplied Tests
+---
+
+## Overview
+
+The Magento Functional Testing Framework Magento supplied Tests are a browser-based user acceptance testing framework used to validate the functionality of an operating Magento site. Running Magento supplied MFTF tests aids EQP QA efforts in assessing critical fucntionality of a Magento instance with the extension installed.
+
+For information on how to setup, create and modify MFTF tests, see [MFTF Devdocs](https://devdocs.magento.com/mftf/docs/introduction.html)
+
+## What testing is for
+
+Magento Supplied MFTF tests validate critical user flows via browser-based testing against an operational Magento instance.
+
+MFTF runs tests in a browser, using Selenium and Codeception, to emulate user behavior. MFTF tests are designed to be extensible and be used in conjunction with other Magento developed MFTF code coverage or your own vendor supplied Tests. The results of this testing will inform QA about the state of various components and functional areas and allow them to prioritize manual testing accordingly.
+
+## When testing is done
+
+MFTF Magento supplied Tests will be run for all Marketplace extension submissions for:
+
+-  Magento in 2.3 and 2.4 release lines
+-  OpenSource and Commerce editions
+
+## What is being checked
+
+Magento supplied MFTF tests are a small set of Critical MFTF tests. These are tests which are part of the Magento MFTF codebase. The list of tests is under review and will be published when confirmed.
+
+## Tools and environments used
+
+The Magento test infrastructure will execute Magento supplied MFTF tests in the most up-to-date version of Magento in the 2.4.x and 2.3.x release lines, as well as the most up-to-date software compatible with that release. The test infrastructure follows the recommended setup for Magento installation and MFTF setup and configuration. You can use Magento Cloud Docker to create a similar environment.
+
+## Reading the error report
+
+-  MFTF returns two types of results
+   -  Simplified results showing the status of each test executed
+   -  Allure results (as XML)
+   -  See [MFTF Reporting](https://devdocs.magento.com/mftf/docs/reporting.html)  for further information
+-  Test failures will be due to either:
+   -  MFTF test flow being broken due to changes to Magento user flows as intended by the extension under test. These can be fixed by using Vendor-Supplied MFTF Tests.
+   -  An failure in Magento user flows due to an unintended side-effect of the extension under test. This may be a type of regression bug and will be tested and verified by manual QA as part of EQP technical review.
```

Comment:
```suggestion
   -  A failure in Magento user flows due to an unintended side-effect of the extension under test. This may be a type of regression bug and will be tested and verified by manual QA as part of EQP technical review.
```